build/transactions/overview.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Overview
+description: An overview of transactions within Quai Network.
+---
+
+# What Are Transactions?
+
+Transactions are cryptographically signed messages sent and executed on Zone chains within Quai Network. Regardless of what ledger or chain you are on within the network, every state-changing operation requires a signed transaction.
```

Comment:
Uupdate first sentence to "Transactions are cryptographically signed messages sent to and executed on Zone chains within Quai Network."